.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+# Golang CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-go/ for more details
+version: 2.1
+
+orbs:
+  aws-cli: circleci/aws-cli@0.1.19
+  k8s: circleci/kubernetes@0.11.0
+
+references:
+  environment: &env
+    GO111MODULE: "on"
+  filters: &branchfilters
+    branches:
+      only: mvp
+
+jobs:
+  build:
```

Comment:
Since this is step is only running unit tests, maybe we can name it `test` or `unit_tests` ?

Reply:
Unfortunately, CircleCI requires a job called "build". :( Yeah, weird, but true..

Reply:
Is there a reason we're using Circle? Why not GH Actions?

Reply:
@jaypipes it is true that CircleCI requires a job called "build" if the `workflows` section isn't defined.

I just tested this, it works fine:
```yaml
version: 2
jobs:
  test:
    docker: 
      - image: alpine
    steps:
      - checkout
      - run: echo "hello world"

workflows:
  version: 2
  test_only:
    jobs:
      - test
```

Reply:
ah, nice catch. leftover knowledge from working with an earlier CircleCI setup, I guess!

Reply:
> Is there a reason we're using Circle? Why not GH Actions?

The CNI and eks-charts repos use CircleCI, so we're copying the work I did for those repos. Also, there are some issues with permissions when using GH Actions. Ask @m00nf1sh for more details :)

Reply:
Thanks! https://github.com/aws/aws-service-operator-k8s/pull/54